docs/CODE_OF_CONDUCT.md

```diff
@@ -53,15 +53,15 @@ Anyone asked to stop unacceptable behavior is expected to comply immediately.
 If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
```

Comment:
Here instead of community organizers, there will be community admins.

Reply:
Done.